game/screens.rpy

```diff
@@ -1,1838 +1,1860 @@
-﻿## Initialization

```

Comment:
What happned to this file? It looks like the entire thing was refreshed?

Guessing there was some odd conflict handling before in this file. We should fix this to preserve history.